frontend/components/ui/registration-card.tsx

```
@@ -0,0 +1,148 @@
+                    Continue with google
+                </button>
+                <div className='account__divider'>
+                    <span>or</span>
+                </div>
+                <form className='account__form' onSubmit={registrationHandler}>
+                    <div className='flex flex-col gap-0 md:gap-5 md:flex-row'>
+                        <div className='w-full'>
+                            <div className='form-grp'>
+                                <label htmlFor='fast-name'>First Name</label>
+                                <input type='text' name='fname' id='fast-name' placeholder='First Name' />
+                                <p className='form_error'></p>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)